jbpm-test-coverage/src/test/java/org/jbpm/test/functional/task/ConcurrentHumanTaskTest.java

```diff
@@ -57,18 +62,33 @@ public ConcurrentHumanTaskTest() {
 		super(true, true);
 	}
 
-	private static final int THREADS = 2;
+	private static final int THREADS = 2;	
+	
+	@Before
+	public void populateOrgEntity() {
+	    TaskService taskService = HumanTaskServiceFactory.newTaskServiceConfigurator().entityManagerFactory(getEmf()).getTaskService();
+	    
+	    ((InternalTaskService)taskService).addUser(TaskModelProvider.getFactory().newUser("krisv"));
+	    ((InternalTaskService)taskService).addUser(TaskModelProvider.getFactory().newUser("sales-rep"));
+	    ((InternalTaskService)taskService).addUser(TaskModelProvider.getFactory().newUser("john"));
+	    ((InternalTaskService)taskService).addUser(TaskModelProvider.getFactory().newUser("Administrator"));
+	    
+	    ((InternalTaskService)taskService).addGroup(TaskModelProvider.getFactory().newGroup("sales"));
+	    ((InternalTaskService)taskService).addGroup(TaskModelProvider.getFactory().newGroup("PM"));
+	    ((InternalTaskService)taskService).addGroup(TaskModelProvider.getFactory().newGroup("Administrators"));
+	}
 
-	@Test
+	@Test(timeout=10000)
```

Comment:
Oops! It's still early morning here. :/ 

Ignore my comment! Deleting it now..